tests/data/example_urls.ris

```diff
@@ -0,0 +1,33 @@
+TY  - JOUR
+AU  - Shannon,Claude E.
+PY  - 1948/07//
+TI  - A Mathematical Theory of Communication
+JF  - Bell System Technical Journal
+SP  - 379
+EP  - 423
+VL  - 27
+UR  - http://example.com
+ER  -
+
+TY  - JOUR
+AU  - Shannon,Claude E.
+PY  - 1948/07//
+TI  - A Mathematical Theory of Communication
+JF  - Bell System Technical Journal
+SP  - 379
+EP  - 423
+VL  - 27
+UR  - http://example.com
+UR  - http://www.example.com
+ER  -
+
+TY  - JOUR
+AU  - Shannon,Claude E.
+PY  - 1948/07//
+TI  - A Mathematical Theory of Communication
+JF  - Bell System Technical Journal
+SP  - 379
+EP  - 423
+VL  - 27
+UR  - http://example.com; http://www.example.com
```

Comment:
A test with `UR  - http://example.com;http://www.example.com` would be nice to have. It works well btw.